patches/graphql-relay+0.7.0.patch

```diff
@@ -0,0 +1,188 @@
+diff --git a/node_modules/graphql-relay/connection/arrayConnection.d.ts b/node_modules/graphql-relay/connection/arrayConnection.d.ts
```

Comment:
Changes made after upgrading to react-relay 0.7.0. They're the same as before but some files moved here and there and I had to adapt to that. Luckily, most changes here are typescript and I think this might be the easier patch that we can get rid of.